src/app/component/common/Card/index.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+import Image from "next/image";
+import { AiOutlineHeart } from "react-icons/ai";
+import { TbCurrencyEthereum } from "react-icons/tb";
+import Button from "../../button/Button";
+
+export type CardProps = {
+  image: string;
+  title: string;
+  time: string;
+  description: {
+    by: string;
+    collateral: string;
+  };
+  bid: {
+    amount: string | number;
+    currency: string;
+  };
+  liked?: boolean;
+  onLike?: () => void;
+  buttonText?: string;
+  onRepay?: () => void;
+  isLliquidation?: boolean;
+  isCurrentBid?: boolean;
+  onButtonClick?: () => void;
+};
+
+const Card = ({
+  image,
+  time,
+  title,
+  description,
+  bid,
+  liked,
+  onLike,
+  onRepay,
```

Comment:
This card is used every where, maybe we can handle this directly from the card via the storage